src/git.zsh

```diff
@@ -170,6 +137,40 @@ _fzf_complete_git() {
     _fzf_path_completion "$prefix" $@
 }
 
+_fzf_complete_git_prefix() {
```

Comment:
It looks like this function is specialized for option prefix. How about renaming this to something like `_fzf_complete_git_option_prefix`?